config/owasp/suppressions.xml

```diff
@@ -37,6 +37,13 @@
    file name: spring-security-crypto-5.1.3.RELEASE.jar
    ]]></notes>
     <gav regex="true">^org\.springframework\.security:spring-security-crypto:5\.1\.3\.RELEASE$</gav>
-    <cpe>cpe:/a:pivotal_software:spring_security</cpe>
+    <cve>CVE-2018-1258</cve>
+  </suppress>
+  <suppress>
+    <notes><![CDATA[
+   file name: spring-security-crypto-5.1.3.RELEASE.jar
+   ]]></notes>
+    <gav regex="true">^org\.springframework\.security:spring-security-crypto:5\.1\.3\.RELEASE$</gav>
+    <cve>CVE-2019-3795</cve>
```

Comment:
Does CVE-2019-3795 need suppressing? I thought the library upgrade stopped the errors for this one?